### DISCUSSION TITLE
error using createArrowFiles()

### DISCUSSION THREAD

**Discussion by sunnyyakima** [GreenleafLab/ArchR#1815] [Questions / Documentation]
I am using a mac
I am following the tutorial. here is the code:

addArchRThreads(threads = 6)
ArrowFiles <- createArrowFiles(
  inputFiles = inputFiles,
  sampleNames = names(inputFiles),
  filterTSS = 4, #Dont set this too high because you can always increase later
  filterFrags = 1000, 
  addTileMat = TRUE,
  addGeneScoreMat = TRUE
)

got a lot of following
2023-01-23 08:12:51 : (scATAC_PBMC_R1 : 3 of 3) Successful creation of Arrow File, 3.965 mins elapsed.
The process has forked and you cannot use this CoreFoundation functionality safely. You MUST exec().
Break on __THE_PROCESS_HAS_FORKED_AND_YOU_CANNOT_USE_THIS_COREFOUNDATION_FUNCTIONALITY___YOU_MUST_EXEC__() to debug.
The process has forked and you cannot use this CoreFoundation functionality safely. You MUST exec().
Break on __THE_PROCESS_HAS_FORKED_AND_YOU_CANNOT_USE_THIS_COREFOUNDATION_FUNCTIONALITY___YOU_MUST_EXEC__() to debug.
.
.
.
The process has forked and you cannot use this CoreFoundation functionality safely. You MUST exec().
Break on __THE_PROCESS_HAS_FORKED_AND_YOU_CANNOT_USE_THIS_COREFOUNDATION_FUNCTIONALITY___YOU_MUST_EXEC__() to debug.
ArchR logging successful to : ArchRLogs/ArchR-createArrows-133c029a89338-Date-2023-01-23_Time-08-08-53.log
Warning message:
In mccollect(jobs) : 3 parallel jobs did not deliver results
> 
> ArrowFiles
NULL

Then, I changed to
addArchRThreads(threads = 1) 

It seems working, but super slow.
Any suggestions? thnaks

**Comment by rcorces** [comment]
In the future, bugs / error reports should be posted to Issues. This looks like an OS / environment problem. As far as I know, we've never seen this before. Googling around seems to indicate that it may be a macOS problem. I dont think there is anything that I can do here. Unless you can prove that parallel computation otherwise works on your machine, I dont think this is related to ArchR.
